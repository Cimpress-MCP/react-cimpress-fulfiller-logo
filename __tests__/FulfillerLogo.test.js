'use strict';

import React from 'react';
import renderer from 'react-test-renderer';

jest.mock('react-visibility-sensor', () => 'div');

import FulfillerLogo from '../src/FufillerLogo.jsx';

describe('FulfillerLogo', function () {

  it('renders correctly', () => {
    const tree = renderer
      .create(<FulfillerLogo/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});