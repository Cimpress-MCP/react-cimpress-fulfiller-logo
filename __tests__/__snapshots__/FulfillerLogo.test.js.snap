// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FulfillerLogo renders correctly 1`] = `
<div
  onChange={[Function]}
  partialVisibility={true}
  scrollCheck={true}
  scrollDelay={1000}
>
  <div
    style={
      Object {
        "alignContent": "center",
        "height": "50px",
        "marginRight": "10px",
        "maxHeight": "50px",
        "maxWidth": "50px",
        "objectFit": "contain",
        "verticalAlign": "middle",
        "width": "50px",
      }
    }
  >
    <span />
  </div>
</div>
`;
